alternator/executor.cc

```diff
@@ -710,6 +710,17 @@ future<executor::request_return_type> executor::list_tags_of_resource(client_sta
     return make_ready_future<executor::request_return_type>(make_jsonable(std::move(ret)));
 }
 
+static future<> wait_for_schema_agreement(db::timeout_clock::time_point deadline) {
+    return do_until([deadline] {
+        if (db::timeout_clock::now() > deadline) {
+            throw std::runtime_error("Unable to reach schema agreement");
```

Comment:
This "throw" means that we fail the table creation, although in this point we know for a fact it *has* been created, I wonder if this won't confuse users. On the other hand, maybe it's fine - a user that will try table creation again thinking it wasn't created, will get a clear error message about the table already existing.